app/models/iiif_content_search_response.rb

```diff
@@ -139,8 +133,21 @@ def after
 
     private
 
+    def highlights
+      # get an array of the starting and ending indices of the <em>-surrounded terms
```

Comment:
Is something like this equivalent?

```
str.scan(%r{<em>.*?</em>})
```

Reply:
I think the method that we're using is trying to be a bit forward thinking about how we'll handle `before`/`after` content in which we could use the the string indexes to get that information.

That being said, I think that supporting `before`/`after` content is something worth discussing given our approach here (and if we wanted to drop that support, we could definitely simplify this by using `str.scan`).

Reply:
Sure -- when we get there, that could get refactored to:

```
str.to_enum(:scan, %r{<em>.*?</em>}).map { |_| Regexp.last_match }
```

Which will be an array of `MatchData` objects with all that information.

Reply:
Neat, I wasn't familiar w/ `MatchData`.  @aeschylus happy to pair up w/ you on this to do a little refactoring.